cli/src/main/java/com/devonfw/tools/ide/tool/ToolCommandlet.java

```diff
@@ -494,6 +272,32 @@ protected void extract(Path file, Path targetDir) {
     }
   }
 
+  /**
+   * @return {@code true} to extract (unpack) the downloaded binary file, {@code false} otherwise.
+   */
+  protected boolean isExtract() {
+
+    return true;
+  }
```

Comment:
if you are moving methods within the same file, it looks like `isExtract()` or `getBinaryName()` has actually been added or removed but actually nothing has changed with these two methods at all.
I know that this quickly happens by accident and you do not notice and see this in your IDE.
However, always watch your diffs before you commit and also have a look into your diff in the PR to see that all is as expected and intended. It is not a big deal if this happens a few time. I just want to raise awareness to make life of reviewers easier in the future :)

Reply:
I will make sure to avoid this in my next PRs :)